src/main/java/teammates/common/util/Const.java

```diff
@@ -1138,24 +1138,31 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         
         public static final String INSTRUCTOR_DETAILS_LENGTH_INVALID = "Instructor Details must have %d columns";
         
-        public static final String STUDENT_FIRST_TIME = "<div style=\"text-align:left;\">Welcome stranger :-) "
-                + "<br><br>It seems you are not a registered user of TEAMMATES. "
-                + "To use TEAMMATES, a course instructor has to add you to a course first. "
-                + "After that, TEAMMATES will send you an email containing the link to 'join' that course. "
-                + "<br><br>Not a stranger to TEAMMATES? "
-                + "Could log in before, but not any more? These are the possible reasons:"
-                + "<br>1. You used a different Google account to access TEAMMATES in the past. "
+        public static final String STUDENT_FIRST_TIME = "<div class='align-left'>"
+                + "<div class='align-center text-color-red text-bold '>"
```

Comment:
Three things:
- Use double-quote for HTML attributes, as how it was done before. Also applies to the line before.
- There's an extra space after `text-bold`.
- To improve the readability of this section, format it like this (e.g. preserve the HTML indentations):
```java
public static final String STUDENT_FIRST_TIME =
          "<div>"
            + "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor "
            + "<div>"
                + "incididunt ut labore et dolore magna aliqua."
            + "</div>"
        + "</div>";
```